src/Web/src/components/layout/Layout.scss

```diff
@@ -1,3 +1,9 @@
+@import "~scss_path/variables";
 .main {
-	height: 93vh;
+	height: $mainHeight;
+	&--flex {
+		flex: 1 1 auto;
+		min-height: $mainHeight;
+		height: initial;
+	}
 }
```

Comment:
To chyba nie potrzebne aż tak to komplikować, inna wysokość jest tylko na homescreen, hotfix powinien naprawić.

Reply:
Prawda, hotfix poprawił. Ja to wczesniej na własną rękę poprawiłem i zostało po mergu.